scripts/cluster.js

```diff
@@ -79,7 +79,7 @@ Cluster.prototype._replicateRawDB = function (sourceDB, targetDB) {
   });
 };
 
-Cluster.prototype._replicateDB = function (sourceDB, targetDB) {
+Cluster.prototype._createdAndReplicateDB = function (sourceDB, targetDB) {
```

Comment:
Could you please rename this from `_createdAndReplicateDB` to `_createAndReplicateDB` (no d)?